logParser.js

```diff
@@ -45,30 +52,37 @@ function parseLog(text, itemList, logServer, client) {
     //test if is auction
     const auction_text = text.match(AUC_REGEX);
     if (auction_text) {
-        client.streamAuction(text.replace(/[|]+/g, '|'), logServer);
         const auction_user = auction_text[1];
         const auction_contents = auction_text[2];
+        client.streamAuction(auction_user, auction_contents.replace(/[|]+/g, '|'), logServer);
```

Comment:
I think we should remove the `replace(/[|]+/g, '|')` here and do sanitization inside the stream function now, as there will be more to do.